extensions/amp-analytics/0.1/vendors/webengage.js

```diff
@@ -17,7 +17,7 @@
 export const WEBENGAGE_CONFIG = /** @type {!JsonObject} */ ({
   'requests': {
     'base':
-      'https://c.webengage.com/amp?licenseCode=${licenseCode}&luid=${clientId(we_luid)}&pageUrl=${canonicalUrl}&pageTitle=${title}&referrer=${documentReferrer}&vh=${viewportHeight}&vw=${viewportWidth}&category=application',
+      'https://c.${region}.webengage.com/amp?licenseCode=${licenseCode}&luid=${clientId(we_luid)}&pageUrl=${canonicalUrl}&pageTitle=${title}&referrer=${documentReferrer}&vh=${viewportHeight}&vw=${viewportWidth}&category=application',
```

Comment:
One question on the `${region}` variable. Is it a mandatory variable? I think the url will break if one fail to specify the value?

Reply:
@zhouyx Yeah the URL will break, if someone doesn't specify that. That's why we have made it a mandatory variable. Is that why the test case is failing?

Reply:
@zhouyx Is there a way to keep the variable optional, such that if it is not specified a fallback/default value would be used in the URL? If not, we'll keep it mandatory.

Reply:
Thanks @ayushgupta29 for clarifying. I'm fine with this as long you make it mandatory. However I know there's no way to actually check the existence of the variable now, so it's really up to you to make sure that the inline config contains the `region` variable. 

>Is there a way to keep the variable optional, such that if it is not specified a fallback/default value would be used in the URL

You could try `$DEFAULT(${region}, default)` in the request. However, there's a known issue that the `$DEFAULT` doesn't work well in variable. We are working on a fix right now. But please test thoroughly : ) 

I'm going to take a look at the failing test case.